test/scrabble_test.rb

```diff
@@ -6,8 +6,20 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    game = Scrabble.new
+
+    assert_equal 1, game.score("a")
+    assert_equal 4, game.score("f")
   end
+
+  def test_can_score_with_multipliers
```

Comment:
I would write a test to see that you can score `""`, `nil`, and `hello` before I started in on the multipliers to confirm that each of those works.